app/analyzers/marc_analyzer.rb

```diff
@@ -7,6 +7,11 @@ def self.accept?(blob)
     MarcRecordService.new(blob).identify != :unknown
   end
 
+  def initialize(blob)
+    super(blob)
+    blob.update_content_type(reader.identify)
```

Comment:
I'm a little wary of doing database updates in the initializer.. but we also seem kinda constrained by the API on offer. I'm going to make a suggestion in `ActiveStorageBlobContentTypes`, but I'm not sure it's any better really..

Reply:
Thanks for the review, and yes I agree with for sure. I've pushed an update.